docs/json_rpc_api.mediawiki

```diff
@@ -2031,6 +2031,13 @@ of the best block.
 : <code>confirmations</code>: <code>(numeric)</code> The number of confirmations.
 : <code>value</code>: <code>(numeric)</code> The transaction amount in DCR.
 : <code>scriptpubkey</code>: <code>(json object)</code> The public key script used to pay coins as a JSON object.
+:: <code>asm</code>: <code>(string)</code> Disassembly of the script.
+:: <code>hex</code>: <code>(string)</code> Hex-encoded bytes of the script. Omitted if empty.
+:: <code>reqSigs</code>: <code>(numeric)</code> The number of required signatures. Omitted if empty.
+:: <code>type</code>: <code>(string)</code> The type of the script (e.g. 'pubkeyhash').
+:: <code>addresses</code>: <code>(json array of strings)</code> The Decred addresses associated with this script. Omitted if empty.
+:: <code>commitamt</code>: <code>(numeric)</code> The ticket commitment value if the script is for a staking commitment. Omitted if empty.
+:: <code>version</code>: <code>(string)</code> The script version.
 : <code>coinbase</code>: <code>(numeric)</code> Whether or not the transaction is a coinbase.
 |-
 !Example Return
```

Comment:
This example return could be updated to include the script version as well.

Reply:
ah

Reply:
Should it be mainnet? I suppose I could just add `,"version": 0`

Reply:
I'd prefer it to be mainnet so it matches the rest of the examples in the doc.  All that really needs to change, imo, from the original example is to add `,"version": 0` as you noted.